test/comparison-tests/create-and-execute-test.js

```diff
@@ -371,6 +371,8 @@ function getNormalisedFileContent(file, location, test) {
                 .replace(/[\d]+[.][\d]* kB/g, ' A-NUMBER-OF kB')
                 // We also don't want a difference in the number of bytes to fail the build
                 .replace(/ \d+ bytes /g, ' A-NUMBER-OF bytes ')
+                // Sometimes "[built]" is written to output, and sometimes not. This should not fail the build
```

Comment:
Oh, and I made a mistake by filtering out `[build]` in stead of `[built]`, but the second integration test of this PR still passed. So there must be something, unrelated to this PR, that makes webpack output `[built]` where it is not expected on seldom occasions, but I am at a loss as to what that something is.